src/main/org/deidentifier/arx/ARXAnonymizer.java

```diff
@@ -544,6 +545,27 @@ private void checkBeforeEncoding(final DataHandle handle, final ARXConfiguration
             if (!definition.getQuasiIdentifiersWithMicroaggregation().isEmpty()) {
                 throw new IllegalArgumentException("Differential privacy must not be combined with micro-aggregation");
             }
+            EDDifferentialPrivacy edpModel = config.getPrivacyModel(EDDifferentialPrivacy.class);
+            if (edpModel.getEpsilon() <= 0d) {
```

Comment:
It would actually make sense to check for correctness in parameterizations in the constructor of the privacy model. However, this is not implemented consistently in ARX, unfortunately. Some models perform checks within their constructors, but many checks are also performed in this method. Consequently, this can remain as-is for now, but please file in issue so that we can fix this for all models in the future.

Reply:
I just filed an issue about this